modules/nodes-pods-configmaps-use-case-consuming-in-volumes.adoc

```diff
@@ -67,7 +67,7 @@ spec:
   containers:
     - name: test-container
       image: gcr.io/google_containers/busybox
-      command: [ "/bin/sh", "cat", "/etc/config/path/to/special-key" ]
+      command: [ "/bin/sh", "-c", "cat", "/etc/config/path/to/special-key" ]
```

Comment:
@rphillips Can you or someone PTAL at these two commands? It appears that I needed to include the `-c` flag for the command to work. Thank you!